### ISSUE TITLE
runtest does not produce log file

### ISSUE THREAD

**Issue by miroi** [bast/runtest#11]
Hi Rado,

I am missing the log-file from test (runtest) run. It's not produces.

~~~
milias@lxir072.gsi.de:~/Work/qch/software/dirac/trunk/test/xyz_symmetry_recognition/../test -v -b ~/Work/qch/software/dirac/trunk/build_intelmkl_i8_xhost/ -l log

running test: test_xyz molecule1
passed

running test: test_xyz molecule2
passed

running test: test_xyz molecule3
passed

running test: test_xyz molecule4
passed

running test: test_xyz molecule5
passed

running test: test_xyz molecule7
passed

running test: test_xyz molecule8
passed
milias@lxir072.gsi.de:~/Work/qch/software/dirac/trunk/test/xyz_symmetry_recognition/.ls -l log
ls: cannot access log: No such file or directory
milias@lxir072.gsi.de:~/Work/qch/software/dirac/trunk/test/xyz_symmetry_recognition/../test --help
Usage: test [options]

runtest 1.3.11 - Numerically tolerant test library.

Options:
  -h, --help            show this help message and exit
  -b BINARY_DIR, --binary-dir=BINARY_DIR
                        directory containing the binary/launcher [default: /u/
                        milias/Work/qch/software/dirac/trunk/test/xyz_symmetry
                        _recognition]
  -w WORK_DIR, --work-dir=WORK_DIR
                        working directory [default: /u/milias/Work/qch/softwar
                        e/dirac/trunk/test/xyz_symmetry_recognition]
  -v, --verbose         give more verbose output upon test failure [default:
                        False]
  -s, --skip-run        skip actual calculation(s) [default: False]
  -d, --debug           print verbose debug information [default: False]
  -l LOG, --log=LOG     log file [default: no logging]
~~~

**Comment by bast** [comment]
I admit that the log function is silly. If you look at the sources you will see that `runtest.py` does not log anything. In your case, the logging is done by `runtest_dirac.py` but the latter only logs if the test fails, not if it passes.

**Comment by bast** [comment]
This will be fixed in the upcoming 2.0.

**Comment by bast** [comment]
Closing this issue since in 2.x (master) this became irrelevant.
